rs/crypto/internal/crypto_lib/threshold_sig/canister_threshold_sig/src/utils/group/secp256k1.rs

```diff
@@ -251,6 +251,19 @@ impl Scalar {
         }
     }
 
+    /// Perform modular inversion
+    ///
+    /// Returns None if no modular inverse exists (ie because the
+    /// scalar is zero)
+    pub fn invert_vartime(&self) -> Option<Self> {
+        let inv = self.s.invert_vartime();
+        if bool::from(inv.is_some()) {
+            Some(Self::new(inv.unwrap()))
+        } else {
+            None
+        }
```

Comment:
I guess
```
self.s.invert_vartime().into_option().map(Self::new)
```
or
```
Option::from(self.s.invert_vartime()).map(Self::new)
```
(both [do not intend to be constant-time](https://docs.rs/subtle/latest/subtle/struct.CtOption.html#note)) would also work.

Same for secp256r1. And possibly also for `invert`.

Reply:
👍